test-project/Assets/EditmodeTests/Subscriptions/SubscriptionAggregateCouplingTests.cs

```diff
@@ -157,45 +140,49 @@ public void DifferentAggregateSubscriptions_should_not_couple_together(Type requ
         // were cached incorrectly would cause the subscriptions to become coupled.
         private void DifferentAggregateSubscriptions_should_not_couple_together_impl<T>() where T : MonoBehaviour
         {
-            var gameObject1 = CreateAndLinkGameObject<T>(1);
-            var gameObject2 = CreateAndLinkGameObject<T>(2);
-
-            RemoveAndUnlinkGameObject(1, gameObject1);
-            RemoveAndUnlinkGameObject(2, gameObject2);
-
-            var gameObject3 = CreateAndLinkGameObject<T>(3);
-            var gameObject4 = CreateAndLinkGameObject<T>(4);
-
-            RemoveAndUnlinkGameObject(3, gameObject3);
-
-            Assert.IsTrue(gameObject4.GetComponent<T>().enabled, "gameObject4.GetComponent<T>().enabled");
+            World
+                .Step(world =>
+                {
+                    world.Connection.CreateEntity(1, GetEntityTemplate());
+                    world.Connection.CreateEntity(2, GetEntityTemplate());
+                })
+                .Step(world =>
+                {
+                    world.CreateGameObject<T>(1);
+                    world.CreateGameObject<T>(2);
+                })
+                .Step(world =>
+                {
+                    world.Connection.RemoveComponent(1, Position.ComponentId);
```

Comment:
This is a workaround for `RemoveEntity` not removing components right?

Reply:
I don't believe its required/matters in this case, just porting the test as close as possible.